lib/features/interactive.js

```diff
@@ -9,24 +9,128 @@ import Promise from 'promise';
 import slugid from 'slugid';
 import SharedFileLock from '../shared_file_lock';
 import url from 'url';
+import wsStream from 'websocket-stream';
+import ws from 'ws';
+import devnull from 'dev-null';
+import streams from 'memory-streams';
+import waitForSocket from '../wait_for_socket';
+import net from 'net';
+import rmrf from 'rimraf';
+import express from 'express';
 
 let debug = Debug('docker-worker:features:interactive');
 
 //Number of attempts to find a port before giving up
 const MAX_RANDOM_PORT_ATTEMPTS = 20;
 
+/** Let displays of a container that /.taskclusterutils is mounted inside */
+let listDisplays = async (container) => {
+  debug('Listing displays');
+  // Run list-displays
+  let exec = await container.exec({
+    AttachStdout: true,
+    AttachStderr: false,
+    AttachStdin: false,
+    Tty: false,
+    Cmd: ['/.taskclusterutils/list-displays']
+  });
+  let stream = await exec.start({
+    Detach: false,
+    stdin:  false,
+    stdout: true,
+    stderr: false,
```

Comment:
Nope... not unless there is a segfault.. .and in that case I just don't care :)

If the output doesn't parse that's just too bad... and we ignore it... This is a best effort service, so we try to fail gracefully..

Note: obviously people can put all sorts of files, folders and domain sockets into `/tmp/.X11-unix/`, and if I try to open those as domain sockets using libX11, I suspect undefined things could happen... So if people intentionally do very obscure things in their container, then absolutely they can build it so that this feature doesn't work.